### PR TITLE
Support for toggling tx confirmation mechanism

### PR DESCRIPTION
**Pull request by adizere**
Closes: #1380

## Description

______

For contributor use:

- [x] Added a changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: relayer-cli/src/commands/tx/packet.rs [hunk 1]
```diff
@@ -37,7 +37,7 @@ impl Runnable for TxRawPacketRecvCmd {
             src_port_id: self.src_port_id.clone(),
             src_channel_id: self.src_channel_id.clone(),
         };
-        let mut link = match Link::new_from_opts(chains.src, chains.dst, opts) {
+        let mut link = match Link::new_from_opts(chains.src, chains.dst, opts, false) {
```
Comment (ancazamfir) [inline relayer-cli/src/commands/tx/packet.rs:40]: why not use the config setting?

**Review by ancazamfir — COMMENTED** [review]
after a quick look, I think that if we hit the periodic clear packet with pending transactions we will submit those again. Maybe @mircea-c could do some eval, maybe in the end it is a small price to pay.

File: relayer-cli/src/commands/tx/packet.rs (hunk 1 above)
Comment (adizere) [reply relayer-cli/src/commands/tx/packet.rs:40]: This is the CLI for doing `tx raw packet-recv`. This means under the hood we're using the `SyncSender` which synchronously waits to confirm transactions. We are not using the `AsyncSender` here, so not sure having tx confirmations would be appropriate.

Comment (ancazamfir) [reply relayer-cli/src/commands/tx/packet.rs:40]: right! missed that one. I hope we move all CLIs and workers to async at some point.

File: relayer/src/worker/client.rs [hunk 2]
```diff
@@ -51,30 +51,37 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> ClientWorker<ChainA, ChainB> {
...
-                        ibc_client_update,
-                        &self.client.dst_chain_id,
-                        &self.client.dst_client_id,
-                        1
-                    );
-                }
-                Err(e) => {
-                    if let ForeignClientErrorDetail::ExpiredOrFrozen(_) = e.detail() {
-                        warn!("failed to refresh client '{}': {}", client, e);
+            if last_refresh.elapsed() > Duration::from_secs(60) {
```
Comment (soareschen) [inline relayer/src/worker/client.rs:60]: Maybe add a comment that explain that we do not need to refresh too frequently.

**Review by soareschen — APPROVED** [review]

**Pull request merged**